src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -68,6 +79,16 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
      ucs_offsetof(uct_cuda_copy_md_config_t, pref_loc),
      UCS_CONFIG_TYPE_ENUM(uct_cuda_pref_loc)},
 
+    {"ALLOC_METHODS", "fabric-device,device",
+     "Priority of allocation types\n"
+     " fabric-device - allocate cuMemCreate pinned device memory with fabric handles\n"
+     " device        - allocate cuMemAlloc pinned device",
+     ucs_offsetof(uct_cuda_copy_md_config_t, alloc_methods), UCS_CONFIG_TYPE_ARRAY(cuda_alloc_methods)},
```

Comment:
we don't need both alloc_methods and alloc_methods configurations. just enable_fabric is enough.

Reply:
@yosefe Is the request to revert the changes from [this](https://github.com/openucx/ucx/pull/9867#discussion_r1596494880) request?

Reply:
IMO, it's ok to leave alloc_methods in case we add another allocation method in the future.